ip-utils.c

```diff
@@ -191,10 +223,32 @@ int janus_network_address_string_buffer_is_null(const janus_network_address_stri
 	return !b || b->family == AF_UNSPEC;
 }
 
-const char * janus_network_address_string_from_buffer(const janus_network_address_string_buffer *b) {
+const char *janus_network_address_string_from_buffer(const janus_network_address_string_buffer *b) {
 	if(janus_network_address_string_buffer_is_null(b)) {
 		return NULL;
 	} else {
 		return b->family == AF_INET ? b->ipv4 : b->ipv6;
 	}
 }
+
+int janus_network_lookup_interface(const struct ifaddrs *ifas, const char *iface, janus_network_address *result) {
+	if(result == NULL)
```

Comment:
You forget to check `ifas` for `NULL`, and do not signal the error explicitly. At the top of the function body, you probably should include something like this:

```
   if(!ifas || !result || !iface) {
      return -EINVAL;
   }
```